7.0-cli/bin/docker-environment

```diff
@@ -18,6 +18,11 @@ chown www-data:www-data $MAGENTO_ROOT
 # Configure PHP
 [ ! -z "${PHP_MEMORY_LIMIT}" ] && sed -i "s/!PHP_MEMORY_LIMIT!/${PHP_MEMORY_LIMIT}/" /usr/local/etc/php/conf.d/zz-magento.ini
 
+[ "$PHP_ENABLE_XDEBUG" = "true" ] && \
+    echo "zend_extension=/usr/local/lib/php/extensions/no-debug-non-zts-20151012/xdebug.so" > \
+        /usr/local/etc/php/conf.d/zz-xdebug-enable.ini && \
+    echo "Xdebug is enabled"
```

Comment:
Can we not use `docker-php-ext-enable` here? I'd prefer that over a hardcoded (dated) path.